cerberus-web/src/main/resources/cerberus.yaml

```diff
@@ -232,6 +232,21 @@ cerberus:
       # Every hour
       cronExpression: "0 0 * ? * *"
 
+    # JWT
+    jwtSecretRefreshJob:
+      enabled: true
+      # Every minute
+      cronExpression: "0 * * ? * *"
+
+    jwtBlacklistCleanUpJob:
```

Comment:
Change this to jwtBlocklistCleanUpJob

Reply:
Also I think this should be disabled by default

Reply:
Won't the database grow without bound though?